i18n/RaidFinder_zh.ts

```diff
@@ -16,17 +16,17 @@
     <message>
         <location filename="../Forms/Tools/DenMap.ui"/>
         <source>Den Map</source>
-        <translation> 巢穴地图</translation>
+        <translation>巢穴地图</translation>
     </message>
     <message>
         <location filename="../Forms/Tools/DenMap.ui"/>
         <source>Wild Area</source>
-        <translation type="unfinished"></translation>
+        <translation>旷野地区</translation>
     </message>
     <message>
         <location filename="../Forms/Tools/DenMap.ui"/>
         <source>Isle of Armor</source>
-        <translation type="unfinished"></translation>
+        <translation>铠之孤岛/translation>
```

Comment:
Missing < to complete the \</translation>.

Reply:
it's ok now